warehouse/query-core/src/main/java/datawave/query/statsd/QueryStatsDClient.java

```diff
@@ -58,19 +58,15 @@ public QueryStatsDClient(String queryId, String host, int port, int maxCacheSize
     /**
      * Get the client, creating it if needed. This should only be called when the clientMonitor has been acquired.
      */
-    private NonBlockingUdpSender client() {
+    private synchronized NonBlockingUdpSender client() {
         if (this.client == null) {
-            synchronized (clientMonitor) {
-                if (this.client == null) {
```

Comment:
Same comment as above about null checks before and after the synchronized call potentially blocking less.